CHANGELOG.md

```diff
@@ -1,5 +1,72 @@
 # Change Log
 
+## v1.6.0 (2021-03-08)
+
+This release adds the capa explorer rule generator plugin for IDA Pro, vivisect support for Python 3 and 12 new rules. We appreciate everyone who opened issues, provided feedback, and contributed code and rules. Thank you also to the vivisect development team (rakuy0, atlas0fd00m) for the Python 3 support (v1.0.0) and the fixes for Python 2 (v0.2.1).

```

Comment:
Lets add a picture of the rule generator plugin here so that people get excited. Maybe also a further explanation below?